AutoscoperM/AutoscoperM.py

```diff
@@ -1027,6 +1042,26 @@ def progressCallback(x):
             filename = os.path.join(outputDir, transformSubDir, segmentName + ".tfm")
             IO.writeTFMFile(filename, spacing, origin)
             self.showVolumeIn3D(segmentVolume)
+
+            bounds = self.GetRASBounds(segmentVolume)
```

Comment:
```suggestion
            bounds = [0] * 6
            segmentVolume.GetRASBounds(bounds)
```

Reply:
Any reason to use this over the static method? I know the `segmentVolume` won't ever be a sequence in this context, so we are unnecessarily checking for that, but the static method is a little cleaner and easier to read (at least to me)
https://github.com/BrownBiomechanics/SlicerAutoscoperM/blob/1667e4378d3526d484affff8a79583b962119a23/AutoscoperM/AutoscoperM.py#L1364-L1371